transformerlab/db.py

```diff
@@ -45,6 +86,14 @@ async def init():
     async with async_engine.begin() as conn:
         await conn.run_sync(models.Base.metadata.create_all)
 
+    # Add trigger_configs column to workflows if it doesn't exist
```

Comment:
Why do we need any of this in the DB? If a workflow is supposed to be triggered by an event, why not just search the list of workflows after every event. That would remove the need for all this stuff in the database entirely.